malasakit-django/feature_phone/fixtures/tl-instructions.yaml

```diff
@@ -0,0 +1,117 @@
+- model: feature_phone.instructions
+  pk: 33
+  fields:
+    key: language-selection
+    language: tl
+    text:
+      To use Filipino, press two.
+- model: feature_phone.instructions
+  pk: 34
+  fields:
+    key: introduction
+    language: tl
+    text:
+      We look forward to listening to your suggestions on how your barangay can
+      better prepare you for natural disasters like typhoons and floods.
+- model: feature_phone.instructions
+  pk: 35
+  fields:
+    key: irb-notice
+    language: tl
+    text:
+      "By answering this, you agree that you are voluntarily taking part in
+      this PCARI Project titled \"E-Participation 2.0: Connecting Diverse
+      Philippine Populations for Disaster Risk Management with a Toolkit
+      Integrating Text and Speech Analytics.\" All of your information and
+      responses gathered through this toolkit will be used as inputs for the
+      Barangay Disaster Preparedness Planning."
+- model: feature_phone.instructions
+  pk: 36
+  fields:
+    key: irb-notice-prompt
+    language: tl
+    text:
+      If you wish to continue, press one; otherwise, press two.
+- model: feature_phone.instructions
+  pk: 37
+  fields:
+    key: irb-notice-exit
+    language: tl
+    text:
+      Thank you for your time and have a nice day.
+- model: feature_phone.instructions
+  pk: 38
+  fields:
+    key: gender-prompt
+    language: tl
+    text:
+      If you are male, press one; if you are female, press two. To skip this
+      question, press the pound sign on your phone's keypad.
+- model: feature_phone.instructions
+  pk: 39
+  fields:
+    key: age-prompt
+    language: tl
+    text:
+      Say your age after the tone.
+- model: feature_phone.instructions
+  pk: 40
+  fields:
+    key: barangay-prompt
+    language: tl
+    text:
+      Which barangay are you from?
+- model: feature_phone.instructions
+  pk: 41
+  fields:
+    key: quantitative-question-instructions
+    language: tl
+    text:
+      Thank you for giving some of your personal information.
+      In a few moments, we will ask you to rate ten statements that are answerable on a scale from one to six.
+      For each statement, record your response after the tone.
+      You may either use your phone's keypad or speak your response.
+      After stating or entering your response from 1 to 6, where 1 represents strongly disagree and 6 represents strongly agree, please wait and you will hear the next statement.
```

Comment:
Since the data entered through Twilio will be from 1-6, can we still merge those data with the existing data from 0-9?

Reply:
No--we will likely create a new (separate) database, since the questions are revamped as well.